docs/Reference.md

```diff
@@ -577,6 +578,31 @@ url
 
 ---
 
+### `onScroll`[⬆](#props-index)<!-- Link generated with jump2header -->
+
+Function that is invoked when the scroll event is fired in the `WebView`.
+
+| Type     | Required | Platform                |
+| -------- | -------- | ----------------------- |
+| function | No       | iOS, macOS, Android, Windows |
+
+Example:
+
+```jsx
+<Webview
+  source={{ uri: 'https://reactnative.dev' }}
+  onScroll={syntheticEvent => {
+    const { contentOffset } = syntheticEvent.nativeEvent
+    console.table(contentOffset)
+  }}
+/>
+```
+
+Function will receive event object of `NativeSyntheticEvent<NativeScrollEvent>`
+type as an argument.
```

Comment:
The documentation for other events also lists the properties of the native event. Could you add that?